build.gradle

```diff
@@ -36,8 +36,6 @@ jenkinsPlugin {
   // Plugin ID, defaults to the project name without trailing '-plugin'
   shortName = 'gradle'
 
-  fileExtension = 'jpi'
```

Comment:
@timja Sorry for introducing this breaking change. Yes, we can revert the extension back to `.hpi` and then release a new version with this change. 

Maybe it's better to specify `fileExtension = 'hpi'` explicitly? To be sure that we don't depend on any implicit behavior of the plugin. WDYT?

Reply:
sure done

Reply:
Thank you! I will release a new version with this change today.